### PR TITLE
OP-269 convert junit tests to assertj

### PR DESCRIPTION
**Pull request by dbmalkovsky**
Convert Junit tests to assertj format
Remove **return;** in void methods

### CODE REVIEW

File: src/test/java/org/isf/opetype/test/TestOperationType.java [hunk 1]
```diff
@@ -57,16 +57,12 @@ public void _setParameters(
...
 	}
 	
 	public void check(
 			OperationType operationType) 
 	{		
-    	assertEquals(code, operationType.getCode());
-    	assertEquals(description, operationType.getDescription());
-		
-		return;
+		assertThat(operationType.getCode()).isEqualTo(code);
```
Comment (emecas) [inline src/test/java/org/isf/opetype/test/TestOperationType.java:65]: ind.

File: src/test/java/org/isf/patvac/test/TestPatientVaccine.java [hunk 2]
```diff
@@ -72,17 +72,13 @@ public void _setParameters(
...
 			PatientVaccine patientVaccine) 
 	{		
-    	assertEquals(lock, patientVaccine.getLock());
-    	assertEquals(progr, patientVaccine.getProgr());
-    	assertEquals(vaccineDate, patientVaccine.getVaccineDate());
-		
-		return;
+    	assertThat(patientVaccine.getLock()).isEqualTo(lock);
+    	assertThat(patientVaccine.getProgr()).isEqualTo(progr);
+    	assertThat(patientVaccine.getVaccineDate()).isEqualTo(vaccineDate);
```
Comment (emecas) [inline src/test/java/org/isf/patvac/test/TestPatientVaccine.java:82]: ind.

File: src/test/java/org/isf/pregtreattype/test/TestPregnantTreatmentType.java [hunk 3]
```diff
@@ -57,16 +57,12 @@ public void _setParameters(
...
 	
 	public void check(
 			PregnantTreatmentType pregnantTreatmentType) 
 	{		
-    	assertEquals(code, pregnantTreatmentType.getCode());
-    	assertEquals(description, pregnantTreatmentType.getDescription());
-		
-		return;
+    	assertThat(pregnantTreatmentType.getCode()).isEqualTo(code);
+    	assertThat(pregnantTreatmentType.getDescription()).isEqualTo(description);
```
Comment (emecas) [inline src/test/java/org/isf/pregtreattype/test/TestPregnantTreatmentType.java:66]: ind.

File: src/test/java/org/isf/sms/test/TestSms.java [hunk 4]
```diff
@@ -74,22 +74,21 @@ public void _setParameters(
...
+    	assertThat(sms.getModuleID()).isEqualTo(moduleID);
+    	assertThat(sms.getSmsDate()).isInSameDayAs(smsDate);
+    	assertThat(sms.getSmsDateSched()).isInSameDayAs(smsDateSched);
+    	if (sms.getSmsDateSent() == null)
+    		assertThat(smsDateSent).isNull();
+    	else
+    	    assertThat(sms.getSmsDateSent()).isInSameDayAs(smsDateSent);
+    	assertThat(sms.getSmsNumber()).isEqualTo(smsNumber);
+    	assertThat(sms.getSmsText()).isEqualTo(smsText);
+    	assertThat(sms.getSmsUser()).isEqualTo(smsUser);
```
Comment (emecas) [inline src/test/java/org/isf/sms/test/TestSms.java:92]: ind.

File: src/test/java/org/isf/therapy/test/TestTherapy.java [hunk 5]
```diff
@@ -82,23 +82,19 @@ public void _setParameters(
...
-		return;
+    	assertThat(therapyRow.getEndDate()).isEqualTo(endDate);
+    	assertThat(therapyRow.getFreqInDay()).isEqualTo(freqInDay);
+    	assertThat(therapyRow.getFreqInPeriod()).isEqualTo(freqInPeriod);
+    	assertThat(therapyRow.getNote()).isEqualTo(note);
+    	assertThat(therapyRow.isNotify()).isEqualTo(notify);
+    	assertThat(therapyRow.getQty()).isEqualTo(qty);
+    	assertThat(therapyRow.isSms()).isEqualTo(sms);
+    	assertThat(therapyRow.getStartDate()).isEqualTo(startDate);
+    	assertThat(therapyRow.getUnitID()).isEqualTo(unitID);
```
Comment (emecas) [inline src/test/java/org/isf/therapy/test/TestTherapy.java:98]: ind.

File: src/test/java/org/isf/vaccine/test/TestVaccine.java [hunk 6]
```diff
@@ -61,16 +61,12 @@ public void _setParameters(
...
 	
 	public void check(
 			Vaccine vaccine) 
 	{		
-    	assertEquals(code, vaccine.getCode());
-    	assertEquals(description, vaccine.getDescription());
-		
-		return;
+    	assertThat(vaccine.getCode()).isEqualTo(code);
+    	assertThat(vaccine.getDescription()).isEqualTo(description);
```
Comment (emecas) [inline src/test/java/org/isf/vaccine/test/TestVaccine.java:70]: ind.

File: src/test/java/org/isf/vactype/test/TestVaccineType.java [hunk 7]
```diff
@@ -57,16 +57,12 @@ public void _setParameters(
...
 	
 	public void check(
 			VaccineType vaccineType) 
 	{		
-    	assertEquals(code, vaccineType.getCode());
-    	assertEquals(description, vaccineType.getDescription());
-		
-		return;
+    	assertThat(vaccineType.getCode()).isEqualTo(code);
+    	assertThat(vaccineType.getDescription()).isEqualTo(description);
```
Comment (emecas) [inline src/test/java/org/isf/vactype/test/TestVaccineType.java:66]: ind.

File: src/test/java/org/isf/visits/test/TestVisit.java [hunk 8]
```diff
@@ -73,18 +73,15 @@ public void _setParameters(
...
 	{		
-    	assertEquals(date,visit.getDate());
-    	assertEquals(note,visit.getNote());
-    	assertEquals(sms,visit.isSms());
-		assertEquals(ward, visit.getWard());
-		assertEquals(patient, visit.getPatient());
-		return;
+    	assertThat(visit.getDate()).isEqualTo(date);
+    	assertThat(visit.getNote()).isEqualTo(note);
+    	assertThat(visit.isSms()).isEqualTo(sms);
```
Comment (emecas) [inline src/test/java/org/isf/visits/test/TestVisit.java:83]: ind.

File: src/test/java/org/isf/ward/test/TestWard.java [hunk 9]
```diff
@@ -76,25 +76,21 @@ public void _setParameters(
...
+    	assertThat(ward.getBeds()).isEqualTo(beds);
+    	assertThat(ward.getDescription()).isEqualTo(description);
+    	assertThat(ward.getDocs()).isEqualTo(docs);
+    	assertThat(ward.getEmail()).isEqualTo(email);
+    	assertThat(ward.getFax()).isEqualTo(fax);
+    	assertThat(ward.isFemale()).isEqualTo(isFemale);
+    	assertThat(ward.isMale()).isEqualTo(isMale);
+    	assertThat(ward.getNurs()).isEqualTo(nurs);
+    	assertThat(ward.isPharmacy()).isEqualTo(isPharmacy);
+    	assertThat(ward.getTelephone()).isEqualTo(telephone);
```
Comment (emecas) [inline src/test/java/org/isf/ward/test/TestWard.java:94]: ind.

**Review by emecas — APPROVED** [review]
Pretty good.  thank you @dbmalkovsky 
I noted the commented problems about indentation, but it is true those could be due to GitHub's visualization. I also know that some tasks are being done on auto styling.

**Review by emecas — APPROVED** [review]
Pretty good.  thank you @dbmalkovsky 
I noted the commented problems about indentation, but it is true those could be due to GitHub's visualization. I also know that some tasks are being done on auto styling.

**Comment by dbmalkovsky** [comment]
I'm not too worried about indentation at this point as the files are currently a combination of space indentation and tab indention.  We have discussed making a single pass through the code and standardizing using the code formatting guidelines pushed earlier for Eclipse and Intellij.

**Pull request merged**